index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+  <!--HEAD-->
+  <head>
+    <meta charset="UTF-8" />
+    <title>Tech Doc | Nisar & Besu</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <!--BODY-->
+  <body>
+    <!--NAVIGATION BAR-->
+    <div id="navbar">
+      <header>JavaScript Documentation</header>
+      <ul>
+        <li><a class="nav-link" href="#Introduction">Introduction</a></li>
+        <li>
+          <a class="nav-link" href="#What_you_should_already_know"
+            >What you should already know
+          </a>
+        </li>
+        <li><a class="nav-link" href="#Variables">Variables</a></li>
+        <li><a class="nav-link" href="#Variable_scope">Variable scope</a></li>
+        <li><a class="nav-link" href="#Constants">Constants</a></li>
+        <li><a class="nav-link" href="#Reference">Reference</a></li>
+      </ul>
+    </div>
+
+    <!--MAIN-->
+    <main id="main-doc">
+      <!--SECTION 1-->
+      <section class="main-section" id="Introduction">
+        <header>Introduction</header>
+        <h1>
+          JavaScript is a cross-platform, object-oriented scripting language. It
+          is a small and lightweight language. Inside a host environment (for
+          example, a web browser), JavaScript can be connected to the objects of
+          its environment to provide programmatic control over them.
+        </h1>
+        <h2>
+          JavaScript contains a standard library of objects, such as Array,
+          Date, and Math, and a core set of language elements such as operators,
+          control structures, and statements. Core JavaScript can be extended
+          for a variety of purposes by supplementing it with additional objects;
+          for example:
+        </h2>
```

Comment:
Good job so far, kindly use p tag instead of using h1 and h2 tags,

The h tags (h1,h2, etc.) are meant for headings and sub-headings, not ordinary text as the <p> (paragraph) tag is.

**Why p tag?** 
Header tags indicate that their contents are more important than normal, and are treated specially by Google's index, and by some text-to-speech browsers as well. Putting long chunks of text inside header tags will confuse Google and possibly get your page penalized in search rank. Use the paragraph tag instead.